install/kubernetes/cilium/charts/agent/templates/clusterrole.yaml

```diff
@@ -55,11 +55,13 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
+  # Deprecated for removal in v1.10
   - create
-  - get
   - list
   - watch
   - update
+
```

Comment:
leftover?

Reply:
No, I separated them to make it clearer which actions are deprecated.